app/helpers/finance_helper.rb

```diff
@@ -1,6 +1,20 @@
 # frozen_string_literal: true
 
 module FinanceHelper
+  MILESTONE_DATES = [
```

Comment:
Is there any logic behind this? Last year of the month for a given set of fixed months?